### PR TITLE
[Nokia][sonic-platform] Update Nokia sonic-platform submodule and device data for thermal logging, etc.

### PR DESCRIPTION
**Pull request by snider-nokia**
These changes, in conjunction with NDK version >= 22.9.17 address the thermal logging issues discussed at https://github.com/Nokia-ION/ndk/issues/27.   While the changes contained at this PR do not require coupling to NDK version >= 22.9.17, thermal logging enhancements will not be available without updated NDK >= 22.9.17.   Thus, coupling with NDK >=22.9.17 is preferred and recommended.

#### Why I did it

To address thermal logging deficiencies.

##### Work item tracking
- Microsoft ADO **(number only)**: 26365734

#### How I did it
The following changes are included:

Threshold configuration values are provided in the associated device data .json files.  There is also a change included to better handle the condition where an SFP module read fails.

Modify the module.py reboot to support reboot linecard from Supervisor
    
     - Modify reboot to call _reboot_imm for single IMM card reboot
     - Add log to the ndk_cmd to log the operation of "reboot-linecard" and "shutdown/satrtup the sfm"

Add new nokia_cmd set command and modify show ndk-status output
    
     - Add a new function reboot_imm() to nokia_common.py to support reboot a single IMM slot from CPM
     - Added new command: nokia_cmd set reboot-linecard <slot> [forece] for CPM
     - Append a new column "RebootStatus" at the end of output of "nokia_cmd show ndk-status"
     - Provide ability for IMM to disable all transceiver module TX at reboot time
     - Remove defunct xcvr-resync service
  
#### How to verify it

Verify that applicable thermal logging is occurring on both LC and Supervisor card.

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [x] 202205
- [ ] 202211
- [x] 202305

#### Tested branch (Please provide the tested image version)

- [x] 202205

#### Description for the changelog

[Nokia][sonic-platform] Update Nokia sonic-platform submodule and device data for enhanced thermal logging  : please couple with NDK >= 22.9.17

### CODE REVIEW

**Comment by snider-nokia** [comment]
@judyjoseph , Can you please review?

File: device/nokia/x86_64-nokia_ixr7250e_sup-r0/platform_ndk.json [hunk 1]
```diff
@@ -31,6 +31,26 @@
...
+      "intval": 3
+    },
+    {
+      "key": "thermal_log_min_threshold",
+      "intval": 5
+    },
+    {
+      "key": "thermal_log_max_threshold",
+      "intval": 1
     }
```
Comment (judyjoseph) [inline device/nokia/x86_64-nokia_ixr7250e_sup-r0/platform_ndk.json:54]: In the **class PublishThermalAlgoAction** in chassis/sonic_platform/thermal_actions.py we do a **return** in execute() function. 

Is there a new different service -- I assume the new sensord implementation which will be doing a UpdateTempInfoPb()?

Comment (snider-nokia) [reply device/nokia/x86_64-nokia_ixr7250e_sup-r0/platform_ndk.json:54]: Yes, the thermal handling is done behind the scenes at our NDK.  Temp info is still updated at Redis DB, but we publish to our Supervisor card directly via NDK mechanism for input to fan control algorithm.

**Review by judyjoseph — APPROVED** [review]

**Comment by gechiang** [comment]
MSFT ADO: 26247825

**Review by judyjoseph — APPROVED** [review]

**Comment by gechiang** [comment]
@StormLiangMS , @yxieca Please help approve for 202305 and 202311.
Thanks!

**Pull request merged**